File: Data Structures & Algorithms/duplicate-integer/submission-2.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
     Arrays.sort(nums);
        for (int i = 0; i < nums.length; i++) {
            if (i > 0 && nums[i] == nums[i - 1]) {
                System.out.println("true");
                return true;
            }
        }
        System.out.println(false);
        return false;
    }
}

File: Data Structures & Algorithms/is-anagram/submission-2.cs
public class Solution {
    public bool IsAnagram(string s, string t) {
        if (s.Length != t.Length) return false;
        
        var sArray = s.ToArray();
        var tArray = t.ToArray();
        Array.Sort(sArray);
        Array.Sort(tArray);
        return sArray.SequenceEqual(tArray);
    }
}
